packages/blocks/embed/src/index.html

```diff
@@ -1,14 +0,0 @@
-<html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-  </head>
-  <body>
-    <div id="app"></div>
-  </body>
-  <script
-    async=""
-    src="https://platform.twitter.com/widgets.js"
-    charset="utf-8"
-  ></script>
```

Comment:
Looks like twitter embedding still works, even without this JS file:
https://blockprotocol-55qywunxr-hashintel.vercel.app/@hash/blocks/embed

<img width="1406" alt="Screenshot 2022-04-21 at 20 19 43" src="https://user-images.githubusercontent.com/608862/164536995-5af64378-77e7-4eba-8c49-113943fc1320.png">